pokemon-api/src/main/java/cz/muni/fi/pa165/pokemon/league/participation/manager/exceptions/InvalidPokemonEvolution.java

```diff
@@ -0,0 +1,27 @@
+package cz.muni.fi.pa165.pokemon.league.participation.manager.exceptions;
+
+/**
+ * Exception for cases when a trainer requests that per Pokemon evolves into
+ * a species that it can't evolve into.
+ * 
+ * @author Tibor Zauko 433531
+ */
+public class InvalidPokemonEvolution extends Exception {
```

Comment:
Again i would rename InvalidPokemonEvolutionException.

Reply:
Done (also I apologize for these stupid mistakes - was tired when creating the exceptions).